index.html.haml

```diff
@@ -94,7 +94,7 @@ desc: Weld - CDI Reference Implementation
               %td
                 %ul
                   %li
-                    Up-to-date version used
+                    %a{:href=>"http://download.jboss.org/weld/3.0.4.Final/wildfly-12.0.0.Final-weld-3.0.4.Final-patch.zip"}Weld 3.0.4.Final
```

Comment:
This does not have the fancy CDI labels attached to it because it realistically supports both on WFLY.